platform/lib/platform_web/live/media_live/search_form.ex

```diff
@@ -0,0 +1,771 @@
+                              </span>
+                            </button>
+                          <% end %>
+                        <% end %>
+                        <%= if !@active_project do %>
+                          <div
+                            data-tooltip="Select a specific project <br/> for more filtering options"
+                            data-tooltip-placement="left"
+                          >
+                            <Heroicons.ellipsis_horizontal class="h-5 w-5 ml-2 py-px inline-block text-neutral-500" />
+                            <span class="text-sm ml-1 leading-normal text-neutral-500">
+                              More Options
+                            </span>
+                          </div>
+                        <% end %>
+                      </div>
+                    </div>
+                    <%= for attr <- @available_attrs do %>
+                      <template x-if={"#{@toggle_state[attr.id] || 'false'}===true && '#{@cur_select}'===\"#{attr.id}\" && '#{@select_state}'==='select_filt'"}>
+                        <div x-transition phx-click-away="select_state_norm" phx-target={@myself}>
+                          <.attr_filter
+                            id={attr.id<>"_dropdown"}
+                            type={:dropdown}
+                            attr_id={Material.MediaSearch.get_attrid(attr.attr)}
+                            form={f}
+                            attr={attr.attr}
+                            _attr={attr}
+                            changeset={@changeset}
+                            is_active={is_active?(@changeset, attr.attr)}
+                            myself={@myself}
+                            default_open={true}
+                          />
+                        </div>
+                      </template>
+                    <% end %>
+                  </article>
+                  <div
+                    class="relative text-left overflow-visible"
+                    data-tooltip="Filter to my assignments"
+                  >
+                    <%= label f, :only_assigned_id, class: "transition-all cursor-pointer flex h-8 border shadow-sm rounded-lg py-1 px-2 w-full justify-center items-center gap-x-1 text-sm text-gray-900 " <> (if Ecto.Changeset.get_field(@changeset, :only_assigned_id) == @current_user.id do
+                      "text-white bg-urge-500 border-urge-500"
+                    else
+                      "bg-white text-neutral-600"
+                    end) do %>
+                      <Heroicons.bookmark solid class="h-5 w-5 py-px" />
+                      <%= checkbox(f, :only_assigned_id,
+                        class: "hidden",
+                        checked_value: @current_user.id
+                      ) %>
+                    <% end %>
+                  </div>
+                  <div
+                    class="relative text-left overflow-visible"
+                    data-tooltip="Filter to my subscriptions"
```

Comment:
Any chance you think we could incorporate these three filter buttons into the new filter menu? (Pending discussion on Discord.)

Reply:
Happy to implement this if we decide on this design!

Reply:
(Decided to keep them separate!)